app/services/invoices/payments/stripe_service.rb

```diff
@@ -89,7 +89,7 @@ def stripe_payment_method
         payment_method = Stripe::PaymentMethod.list(
           {
             customer: customer.stripe_customer.provider_customer_id,
-            type: 'card', # TODO: Supported payment method type
+            # type: 'card', # TODO: Supported payment method type
```

Comment:
Don't we have to keep sending the payment method?

Reply:
I think we should skip the type here, so that we list also other payment method than `card`. We need also `sepa_debit`.